helm/hpcc/templates/_helpers.tpl

```diff
@@ -160,7 +171,10 @@ storage:
   - name: {{ $plane.name | quote }}
   {{- $planeYaml := omit $plane "name" "pvc" "storageClass" "storageSize" "subPath" -}}
   {{- if $plane.subPath -}}
-   {{- $_ := set $planeYaml "prefix" (printf "%s/%s" $planeYaml.prefix $plane.subPath) -}}
+    {{- $_ := set $planeYaml "prefix" (printf "%s/%s" $planeYaml.prefix $plane.subPath) -}}
+  {{- end -}}
+  {{- if and (eq "data" $plane.category) (not $plane.defaultSprayParts) -}}
+    {{- $_ := set $planeYaml "defaultSprayParts" (include "hpcc.getMaxNumWorkers" $) -}}
```

Comment:
One minor comment.  This generates the number of spray parts as a quoted string, rather than an integer.  Debatable whether it is worth cleaning up or not, probably yes if also fixing the indentation.